glide.lock

```diff
@@ -263,27 +236,75 @@ imports:
   version: 3887ee99ecf07df5b447e9b00d9c0b2adaa9f3e4
 - name: gopkg.in/yaml.v2
   version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
+- name: k8s.io/apimachinery
```

Comment:
it sure was nice before when everything we pulled in from k8s was from the client-go repo. Seems odd that they added this dependency, but i must not understand it yet